force-app/main/default/classes/AccountModelSettingsVMapper_TEST.cls

```diff
@@ -138,10 +135,158 @@ public with sharing class AccountModelSettingsVMapper_TEST {
         Test.stopTest();
 
         System.assertEquals(
-            RecordTypeService.getInstance(), 
-            recordTypeServiceInstance, 
+            RecordTypeService.getInstance(),
+            recordTypeServiceInstance,
             'Should return the instance of RecordTypeService'
         );
     }
-}
 
+    /***********************************************************************************************************************************
+     * @description Test method to verify that the getAccountModelSettingsViewModel returns an AccountModelSettingsVModel with the selected
+     * value set to the current field value specified in Hierarchy Settings for Account Record Types and a list of comboboxVModels that
+     * correspond to all active Account Record Types in the org.
+     ***********************************************************************************************************************************/
+    @isTest
+    private static void getAccountModelSettingsViewModelValid() {
+        RecordTypeMapper.instance = new STUB_RecordTypeMapper();
+
+        Hierarchy_Settings__c hierarchySettings = UTIL_CustomSettings_API.getSettingsForTests(
+            new Hierarchy_Settings__c(
+                Account_Processor__c = '000000000000000001',
```

Comment:
That's a clever way to use hybrid settings to get everything you need out of one mapper.